scapy/utils.py

```diff
@@ -971,6 +971,10 @@ def __call__(cls, filename):
         """
         i = cls.__new__(cls, cls.__name__, cls.__bases__, cls.__dict__)
         filename, fdesc, magic = cls.open(filename)
+        if not magic:
+            raise Scapy_Exception(
+                "The file descriptor is closed !"
```

Comment:
Something like that maybe?
```suggestion
                "No data could be read!"
```